### DISCUSSION TITLE
Feature Request: Show percentage change from this period to the previous period

### DISCUSSION THREAD

**Discussion by uptickmetachu** [grafana/grafana#72916] [Dashboards, Panels & Visualizations]
<img width="302" alt="image" src="https://github.com/grafana/grafana/assets/67882333/c8747886-74a1-451c-bef5-d47d70681b20">

It would be useful to have a stat panel that can show the percentage difference between 2 values and on hover, a tooltip showing the absolute difference between the two values.

This is commonly used in reporting dashboards where single stats such as "total web site viewers over the last 30 days" can be compared against the previous period of 30 days. 

This would be a concise way of capturing the difference between two time periods without showing a time series panel.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82544. Feel free to continue the discussion around this there. Thank you!
